Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer_README.md

```diff
@@ -27,15 +27,15 @@ CVE-2025-31324 is actively exploited and is critically severe. Organizations sho
 
 [View official CVE details on NIST](https://nvd.nist.gov/vuln/detail/CVE-2025-31324)
 
-##  Alert Triggering
-1. Manually
-2.  "CVE Exploitation - 986328356" Agent rule
+##  Playbook Triggers
+- Manually
+- "CVE Exploitation - 986328356" Agent rule
 
- ##  Playbook Flow
+##  Playbook Flow
 1. Using XQL, identify potential SAP NetWeaver instances in your environment.
 2. Using XQL, check if there are events that point to any potential webshells downloaded in the directories.
 3. Directory enumeration to identify if there are already any suspicious files that might indicate a webshell.
-4. *Collect* IOCs from a Unit42 blog, *search* them using the "Threat Hunting - Generic" playbook (supports Palo Alto networks products, Qradar, and Splunk), and *block* those indicators
+4. *Collect* IOCs from a Unit42 blog, *search* them using the "Threat Hunting - Generic" playbook ( supports Palo Alto networks products, Qradar, and Splunk), and *block* those indicators using the "Containment Plan - Block Indicators" playbook.
```

Comment:
appears to be an unintentional space inside the parentheses before 'supports'

Reply:
Done